test/runtime/samples/store-assignment-updates-destructure/main.svelte

```diff
@@ -0,0 +1,34 @@
+<script>
+	import { writable } from 'svelte/store';
+
+	let userName1 = writable('init1')
+	let userName2 = writable('init2')
+	let userName3 = writable('init3')
+	let userName4 = writable('init4')
+	let userName5 = writable('init5')
+	let userName6 = writable('init6')
+	let userName7 = writable('init7')
+	
+	let obj = {
+		userName1: 'user1', 
+		userName2: 'user2', 
+		userName3: 'user3', 
+		$userName4: 'user4',
+		userName5: 'user5',
+		$userName6: 'user6',
+		userName7: 'user7',
+	}
+	
+	;({userName1: $userName1, $userName2 } = obj)
```

Comment:
this seems like strange placement of a semicolon. any reason to start the line with it instead of ending the previous line with it?

Reply:
ah probably i just copied the code from the issue's repro